Tools/.gitignore

```diff
@@ -1,6 +1,8 @@
 nuget.exe
 NUnit.*
```

Comment:
Ii should be changed to `nunit.*`. `NUnit` was the right case prior to this PR, but now the NUnit directories are in lower case. It is not an issue under Windows, but it is one under Linux. It causes all those NUnit packages to be seen as untracked.

Reply:
Changed and also removed `SQLitePCLRaw.core.*` as it is not needed anymore.